cirq-core/cirq/transformers/gauge_compiling/gauge_compiling_test_utils.py

```diff
@@ -73,6 +74,45 @@ def test_all_gauges(self, mock_select, seed):
             else:
                 _check_equivalent_with_error_message(c, nc, gauge)
 
+    def test_sweep(self):
+        a = cirq.NamedQubit('a')
```

Comment:
add 
```py3
if not self.enable_test_sweep: return
```
to the start of the test

Reply:
updated the param name to be sweep_must_pass, for those not implemented, expect raising not implemented error, thus all gauge transformer will be tested instead of return at the beginning.

after all gauges are implemented we can remove this variable.